.yarnrc.yml

```diff
@@ -1,4 +1,4 @@
-yarnPath: ".yarn/releases/yarn-berry.cjs"
+# note setting yarnPath as yarn 3 is provided by nix shell
```

Comment:
I wonder if this can be problamatic for people not using nix, since they'll need to manually set a yarPath to the berry release

Reply:
yes, this would be a problem

however I'd say it's fair to require that people install `nix` if the want to play with the source code as this way we don't need to ask them to install anything else on their system

people who want to play with the module itself can install it via npm in which case nix is not required